samples/js/node/notebooks/004-hello-detection-with-output.nnb

```diff
@@ -0,0 +1,161 @@
+{
+    "cells": [
+        {
+            "language": "markdown",
+            "source": [
+                "# Hello Object Detection\n\nA very basic introduction to using object detection models with OpenVINO™.\n\nThe [horizontal-text-detection-0001](https://docs.openvino.ai/2023.0/omz_models_model_horizontal_text_detection_0001.html) model from [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/) is used. It detects horizontal text in images and returns a blob of data in the shape of `[100, 5]`. Each detected text box is stored in the `[x_min, y_min, x_max, y_max, conf]` format, where the\n`(x_min, y_min)` are the coordinates of the top left bounding box corner, `(x_max, y_max)` are the coordinates of the bottom right bounding box corner and `conf` is the confidence for the predicted class."
+            ],
+            "outputs": []
+        },
+        {
+            "language": "markdown",
+            "source": [
+                "## Imports"
+            ],
+            "outputs": []
+        },
+        {
+            "language": "typescript",
+            "source": [
+                "const cv2 = require('opencv.js');\nconst { transform, getImageData, displayArrayAsImage, downloadFile } = require('./helpers.js');\nconst ov = require('../node_modules/openvinojs-node/build/Release/ov_node_addon.node');"
+            ],
+            "outputs": []
+        },
+        {
+            "language": "markdown",
+            "source": [
+                "## Download the Model"
+            ],
+            "outputs": []
+        },
+        {
+            "language": "typescript",
+            "source": [
+                "const baseArtifactsDir = '../../assets/models';\n\nconst modelName = 'horizontal-text-detection-0001';\nconst modelXMLName = `${modelName}.xml`;\nconst modelBINName = `${modelName}.bin`;\n\nconst modelXMLPath = baseArtifactsDir + '/' + modelXMLName;\n\nconst baseURL = 'https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.3/models_bin/1/horizontal-text-detection-0001/FP32/';\n\ndownloadFile(baseURL + modelXMLName, modelXMLName, baseArtifactsDir);\ndownloadFile(baseURL + modelBINName, modelBINName, baseArtifactsDir);"
+            ],
+            "outputs": []
+        },
+        {
+            "language": "markdown",
+            "source": [
+                "## Load the Model"
+            ],
+            "outputs": []
+        },
+        {
+            "language": "typescript",
+            "source": [
+                "const modelPath = '../../assets/models/horizontal-text-detection-0001.xml';\nconst core = new ov.Core();\nconst model = core.read_model(modelPath);\nconst compiledModel = core.compile_model(model, 'CPU');\n\nconst outputLayer = compiledModel.outputs[0];\nconst inputLayer = compiledModel.inputs[0];"
```

Comment:
A suggestion, here we can  use method input(index) and output("boxes") to show more functionalities, e.g that we can access nodes using indexes and strings
`const inputLayer = compiledModel.input(0);
const outputLayer = compiledModel.output("boxes");`